notebooks/02_dj_webinar.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<p>Would it be good to plot an ERD here? Just to remind them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/datajoint-webinar/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Agreed and changed.</p>